ext/project_data.yaml

```diff
@@ -34,22 +34,24 @@ gem_platform_dependencies:
     gem_runtime_dependencies:
       # Pinning versions that require native extensions
       ffi: '~> 1.9.6'
-      win32-dir: '~> 0.4.9'
-      win32-eventlog: '~> 0.6.2'
-      win32-process: '~> 0.7.4'
+      # win32-xxxx gems are pinned due to PUP-6445
+      win32-dir: '= 0.4.9'
+      win32-eventlog: '= 0.6.5'
```

Comment:
After reviewing changes:

* https://github.com/djberg96/win32-eventlog/compare/win32-eventlog-0.6.2...win32-eventlog-0.6.5 - 0.6.5 is fine
* https://github.com/djberg96/win32-process/compare/win32-process-0.7.4...win32-process-0.7.5 - 0.7.5 is fine
* https://github.com/djberg96/win32-service/compare/win32-service-0.8.6...win32-service-0.8.7 - 0.8.7 is fine